solr/core/build.gradle

```diff
@@ -152,9 +125,10 @@ dependencies {
   implementation 'org.apache.httpcomponents:httpclient'
   implementation 'org.apache.httpcomponents:httpcore'
 
-  compileOnlyApi 'org.eclipse.jetty.toolchain:jetty-servlet-api'
-
   implementation 'org.eclipse.jetty:jetty-client'
+  implementation 'org.eclipse.jetty:jetty-http'
+  implementation 'org.eclipse.jetty:jetty-io'
+  implementation 'org.eclipse.jetty.toolchain:jetty-servlet-api'
```

Comment:
maybe this should be api? I'm not sure. Thoughts @risdenk ?

Reply:
I think `implementation` is fine for now. I can double check tomorrow.

Reply:
yea implementation is good for now. I don't see a reason to make it api.